frontend/src/pages/articles/components/articles-list/articles-list.tsx

```diff
@@ -19,24 +20,32 @@ const ArticlesList: React.FC<Properties> = ({
   articles,
   isLoading,
   onFetchData,
-}) => (
-  <InfiniteScroll
-    hasMore={hasMore}
-    className={styles.articles}
-    dataLength={articlesLength}
-    isLoading={isLoading}
-    fetchData={onFetchData}
-  >
-    {articles.map((article) => (
-      <ArticleCard
-        key={article.id}
-        article={article}
-        author={article.author}
-        tags={getArticleTags(article)}
-        reactions={article.reactions}
-      />
-    ))}
-  </InfiniteScroll>
-);
+}) => {
+  if (!isLoading && !articles.length) {
+    return <EmptyArticlesPlaceholder />;
+  }
+
+  return (
+    <Loader isLoading={isLoading && !articles.length} type="circular">
+      <InfiniteScroll
+        hasMore={hasMore}
+        className={styles.articles}
+        dataLength={articlesLength}
+        isLoading={isLoading}
+        onFetchData={onFetchData}
+      >
```

Comment:
Inside InfiniteScroll - there is a Loader passed through - isn't it suitable?

Reply:
I am about loader wrapping Infinite Scroll

Reply:
At first I've tried without the loader, but for some reason the loader doesn't display on the first fetch

Reply:
Okay, thanks